### PR TITLE
Jmanko/onem 20073

### PR DESCRIPTION
**Pull request by jacek-manko-red**

### CODE REVIEW

**Comment by ghost** [comment]
Looks good to me.

File: Source/WebKit/StorageProcess/StorageProcess.cpp [hunk 1]
```diff
@@ -310,11 +310,13 @@ void StorageProcess::fetchWebsiteData(PAL::SessionID sessionID, OptionSet<Websit
 {
     auto websiteData = std::make_unique<WebsiteData>();
     WebsiteData* rawWebsiteData = websiteData.get();
+    UNUSED_PARAM(rawWebsiteData);
```
Comment (jacek-skiba-red) [inline Source/WebKit/StorageProcess/StorageProcess.cpp:313]: remove rawWebsiteData

File: Source/cmake/OptionsWPE.cmake [hunk 2]
```diff
@@ -235,6 +235,8 @@ if (COMPILER_IS_GCC_OR_CLANG AND UNIX AND NOT APPLE)
         set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -fno-stack-protector")
         set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -fno-stack-protector")
     endif()
+    set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -Werror -Wformat=1 -Wno-pragmas")
```
Comment (jacek-skiba-red) [inline Source/cmake/OptionsWPE.cmake]: why not move to previous line which sets C/C++ compiler flags?

File: Source/JavaScriptCore/runtime/IntlNumberFormat.cpp [hunk 3]
```diff
@@ -475,6 +475,10 @@ ASCIILiteral IntlNumberFormat::partTypeString(UNumberFormatFields field, double
     case UNUM_EXPONENT_FIELD:
 #if !defined(U_HIDE_DEPRECATED_API)
     case UNUM_FIELD_COUNT:
+#endif
+#if !defined(U_HIDE_DRAFT_API)
```
Comment (jacek-skiba-red) [inline Source/JavaScriptCore/runtime/IntlNumberFormat.cpp:479]: what for is this define?

File: Source/WebKit/StorageProcess/StorageProcess.cpp (hunk 1 above)
Comment (jacek-manko-red) [reply Source/WebKit/StorageProcess/StorageProcess.cpp:313]: can't remove it - it's used in #if ENABLE(SERVICE_WORKER) or #if ENABLE(INDEXED_DATABASE) blocks, see below

File: Source/JavaScriptCore/runtime/IntlNumberFormat.cpp (hunk 3 above)
Comment (jacek-manko-red) [reply Source/JavaScriptCore/runtime/IntlNumberFormat.cpp:479]: It's from ICU, used in ICU header that we include, if undefined then those 2 additional "draft" enum values are visible

File: Source/cmake/OptionsWPE.cmake (hunk 2 above)
Comment (jacek-manko-red) [reply Source/cmake/OptionsWPE.cmake]: OK I'll move it there

**Review by jacek-skiba-red — APPROVED** [review]

**Pull request merged**